README.md

```diff
@@ -1 +1,131 @@
 # Hub Chain PoC
+
+Proof-of-Concept implementation of a Hub Chain based on Hyperledger Iroha 2.
+
+The core idea of a hub chain is to move assets across Iroha 2 blockchains (called _domestic chains_) using a **central hub chain**.
+
+The transfer is split in two steps:
+
+1. Chain 1 → Hub Chain
+2. Hub Chain → Chain 2
+
+Each step involves a **prover** (source chain) and a **verifier** (target chain). However, for now we will be implementing a simple flow without proofs.
+
+## Stage 1 _(you are here)_: No-proof
+
+In this version, there is no prover-verifier mechanism. Cross-chain transfers are done by the trusted relays.
+
+This lacks the core requirement for the hub chain, but may be a good preparation to introduce a prover-verifier mechanism later based on this work (even if there will be drastic design changes).
+
+### Design
+
+- There are $N$ domestic chains and 1 hub chain,
+- Each _domestic_ chain has an arbitrary amount of user accounts, $N - 1$ _omnibus_ accounts, and 1 _relay_ account
+- Hub chain has $N$ omnibus accounts and $N$ relay accounts
+- _All_ chains have the same numeric asset definitions (e.g. `rose#wonderland`, `time#looking_glass` etc)
+- Balances of omnibus accounts are representing totals of their associated chain in order. If some asset is minted on some chain, it must be minted to the omnibus account of this chain on all other chains (both domestic and the hub).
+- Transfers are expressed as native `Transfer` instructions. To make a **cross-chain transfer**, user must transfer to an omnibus account of the target chain and attach metadata with that chain's destination account `destination: AccountId`. Then, relays forward this transfer through the hub to the destination chain and account.
+
+### Example accounts layout
+
+- Chain `A`
+  - Account 1
+  - Account 2
+  - Account 3
+  - Omnibus Account `B` (e.g. `omni@chain_B`)
+  - Omnibus Account `C`
+  - Relay `A`
+- Chain `B`
+  - Account 1
+  - Account 2
+  - Account 3
+  - Relay `B`
+  - Omnibus Account `A`
+  - Omnibus Account `C`
+- Chain `C`
+  - _...user accounts..._
+  - Relay `C`
+  - Omnibus Account `A`
+  - Omnibus Account `B`
+- Hub Chain
+  - Relay `A`
+  - Relay `B`
+  - Relay `C`
+  - Omnibus Account `A`
+  - Omnibus Account `B`
+  - Omnibus Account `C`
+
+### Transfer flow example
+
+Transfer `rose##alice@wonderland` (i.e. asset `rose#wonderland` from account `alice@wonderland`) of quantity 42 on chain `aaa` to account `mad_hatter@looking_glass` on chain `bbb`:
+
+1. On Chain `aaa` (by user): `Transfer`
+   - `object=rose##alice@wonderland`
+   - `target=omni@chain_bbb`
+   - `quantity=42`
+   - `metadata(destination=mad_hatter@looking_glass)`
+2. On Hub Chain (by relay of chain `aaa`): `Transfer`
+   - `object=rose#wonderland#omni@chain_aaa`
+   - `target=omni@chain_bbb`
+   - `quantity=42`
+   - `metadata(destination=mad_hatter@looking_glass)`
```

Comment:
To convey information about the remittance source, the metadata also needs to include `source`.